packages/buttons/stories/2-Button.stories.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Buttons/Button',
+  component: Button
+} as Meta;
+
+export const Default: Story = ({
+  isDanger,
+  size,
+  isStretched,
+  isPrimary,
+  isBasic,
+  isLink,
+  isPill,
+  focusInset
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <Button
```

Comment:
Please add a control for `disabled`. Also, it would be good to allow the user to enter text. See https://zendeskgarden.github.io/react-components/buttons/#basic for details. (Also, the knobs ordering of the styleguidist example is preferable.)